projects/hslayers/src/components/utils/utils.service.ts

```diff
@@ -7,6 +8,11 @@ import {isPlatformBrowser} from '@angular/common';
 
 @Injectable()
 export class HsUtilsService {
+  queues: {
```

Comment:
Not sure if it should go to utils. Queue management should be a seperate module in common directory imho